src/Windows/Avalonia.Win32/Avalonia.Win32.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\packages\Avalonia\Avalonia.csproj" />
     <PackageReference Include="Avalonia.Angle.Windows.Natives" Version="2.1.0.2019013001" />
+    <PackageReference Include="Microsoft.Windows.SDK.NET" Version="10.0.18362.3-preview" />
```

Comment:
Just a heads up, the mechanism for this is going to change before .NET 5 and we're going to have to cross target between netstandard2.0 (and use the old system) and net5.0 (and use the new system, which is still going through significant changes)

Reply:
It will still be possible to run that netstandard2.0 code against net50, right?

Reply:
No. The netstandard2.0 code will use the old projection system, and the net5.0 code will use the new system which is built on new APIs only available in net5.0

Reply:
The netstandard2.0 target of Microsoft.Windows.SDK.NET to my knowledge will not be available at the final release.

Reply:
So we can just copy-paste the required parts and make them internal, right?

Reply:
The embedded scenario still needs some work to finish it up and is not recommended for use unless absolutely required, but it could work as long as the WinRT objects are never exposed via a public API.